pkg/storage/mto_category.go

```diff
@@ -82,6 +82,17 @@ func MTOCategoryCreate(np sqlutils.NamedPreparer, _ *zap.Logger, MTOCategory *mo
 	return returned, nil
 }
 
+// MTOCategoryDelete deletes an existing MTOCategory from the database
+func MTOCategoryDelete(np sqlutils.NamedPreparer, _ *zap.Logger, id uuid.UUID) error {
+	arg := map[string]interface{}{"id": id}
```

Comment:
See MTOMilestoneDelete,

You need to set a transaction level variable of the user_id when deleting objects from the database in order to track  auditing information
```go
	err := setCurrentSessionUserVariable(tx, actorUserID)
	if err != nil {
		return err
	}
```